mokapot/model.py

```diff
@@ -592,30 +568,36 @@ def _get_starting_labels(psms, model):
 
     elif model.is_trained:
         try:
-            scores = model.estimator.decision_function(psms.features)
+            scores = model.estimator.decision_function(psms.features.values)
         except AttributeError:
```

Comment:
@ezander please comment

Reply:
Good question! This pull request did not introduce the error handling though - see the main branch https://github.com/wfondrie/mokapot/blob/main/mokapot/model.py#L594-L599 

I would speculate that this might handle different classifier used for the percolator model if sometimes the decision function might not be implemented in sklearn for the classifier. Maybe @wfondrie can comment on this fallback from sklearns `decision_function` to `predict_proba`.

Reply:
https://github.com/wfondrie/mokapot/pull/119#issuecomment-2231943971 <- Added there to the series of things that should be done but not now